src/Mailjet/Message/CommitteeCitizenInitiativeNotificationMessage.php

```diff
@@ -79,37 +76,28 @@ public static function create(array $recipients, CommitteeFeedItem $feedItem, st
     }
 
     private static function getTemplateVars(
+        string $recipientFirstName,
```

Comment:
There is no need of this variable here, isn't it ?

Reply:
good catch